docs/reference/errors-and-warnings/NU1604.md

```diff
@@ -12,10 +12,49 @@ f1_keywords:
 
 # NuGet Warning NU1604
 
+## Missing Package Version
+
+> Project dependency 'PackageA' does not contain an inclusive lower bound. Include a lower bound in the dependency version to ensure consistent restore results.
+
+### Issue
+
+A project dependency doesn't define a version.
+This means that restore used the lowest available version.
+Each restore will float downwards trying to find a lower version that can be used.
+This means that restore goes online to check all sources each time instead of using the packages that already exist in the user package folder.
+
+### Solution
+
+Find the `PackageReference` item that does not define the `Version` attribute and add it:
+
+For example change from:
+
+> `<PackageReference Include="PackageA" />`
+
+to:
+
+> `<PackageReference Include="PackageA" Version="9.0.0" />`
+
+If the project is using [NuGet's Central Package Management (CPM)](../../consume-packages/Central-Package-Management.md), you need to update the `<PackageVersion />` item in `Directory.Packages.props` and change from:
+
+> `<PackageVersion Include="PackageA" />`
+
+to:
+> `<PackageVersion Include="PackageA" Version="9.0.0" />`
+
+If a version is specified in a `<PackageVersion />` item and you still receive this warning, verify you've correctly [onboarded to central package management](../../consume-packages/Central-Package-Management.md#enabling-central-package-management).
+
+> [!Note]
+> When using CPM and the file `Directory.Packages.props` is invalid, NU1604 is raised.
+
+## Missing Inclusive Lower Bound
+
 > Project dependency 'PackageA' (&lt;= 9.0.0) does not contain an inclusive lower bound. Include a lower bound in the dependency version to ensure consistent restore results.
 
 ### Issue
-A project dependency doesn't define a lower bound.<br/><br/>This means that restore did not find the *best match*. Each restore will float downwards trying to find a lower version that can be used. This means that restore goes online to check all sources each time instead of using the packages that already exist in the user package folder.
+A project dependency doesn't define a lower bound.
```

Comment:
there needs to be one empty line for markdown to render as a different paragraph. It's not necessary to use a single `<br/>`. 

There's no way to have markdown render a double length gap between paragraphs, but I don't know why that would be desirable.

Reply:
GitHub's showing there is no empty line, see the line numbers 55, 56, 57.  Since the old file was one long line, not shown in the screenshot,  line 55 is line wrapped, making line 55 look like there's a gap, which is why checking the line numbers is important.

Reply:
The rendering looks good so I'll resolve this conversation.